python/rmm/_cuda/stream.pyx

```diff
@@ -94,14 +94,17 @@ cdef class Stream:
         return self.c_is_default()
 
     def _init_from_numba_stream(self, obj):
-        self._cuda_stream = <cudaStream_t>(obj.handle.value)
+        if obj.handle.value == None:
```

Comment:
When comparing with `None`, use `is`, since there's only ever one `None` object :)

```suggestion
        if obj.handle.value is None:
```

Reply:
Nice catch, thanks @shwina !